RocketData/BatchDataProviderListener.swift

```diff
@@ -24,23 +24,23 @@ import ConsistencyManager
  In general, you probably don't need to worry about this; it's just recommended you don't try to batch listen on every data provider in your app.
  The intended use case is when you have a view controller which has multiple data providers and wants to update in just one place.
  */
-public class BatchDataProviderListener: BatchListenerDelegate {
+open class BatchDataProviderListener: BatchListenerDelegate {
 
     /// You must assign a delegate if you want a callback when one of the data providers gets updated
-    public weak var delegate: BatchDataProviderListenerDelegate?
+    open weak var delegate: BatchDataProviderListenerDelegate?
 
     /// The consistency manager which is backed by this instance
-    public let consistencyManager: ConsistencyManager
+    open let consistencyManager: ConsistencyManager
 
     /// This is the batch listener from the consistency manager which contains most of the logic for doing batch listening
-    private let batchListener: BatchListener
+    fileprivate let batchListener: BatchListener
```

Comment:
Not a pain. Pretty easy.
Sort of a funny task though because I'll just change them back if I need to.